src/utils/constants/dust.ts

```diff
@@ -20,19 +20,29 @@ export const DUST_GRADE: { [key: number]: GradeType } = {
   4: 'DANGER',
 };
 
+export const INIT_DUST_SCALE = 0;
+export const INIT_DUST_GRADE = 0;
+
+export const INIT_DUST_INFO = {
+  fineDustScale: 0,
+  fineDustGrade: 0,
+  ultraFineDustScale: 0,
+  ultraFineDustGrade: 0,
+};
+
```

Comment:
```suggestion
export const INIT_DUST_SCALE = 0;
export const INIT_DUST_GRADE = 0;

export const INIT_DUST_INFO = {
  fineDustScale: INIT_DUST_SCALE,
  fineDustGrade: INIT_DUST_GRADE,
  ultraFineDustScale: INIT_DUST_SCALE,
  ultraFineDustGrade: INIT_DUST_GRADE,
};

export const INIT_SIDO_DUST_INFO = {
  location: '정보를 불러오지 못했어요',
  ...INIT_DUST_INFO,
}

export const INIT_CITY_DUST_INFO = {
  location: '정보를 불러오지 못했어요',
  dataTime: INIT_DATA_TIME,
  ...INIT_DUST_INFO,
}
```

이렇게 작성하면 하드코딩을 줄일 수 있을 것 같아요!